HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -453,17 +453,57 @@ public async Task<IActionResult> Eligible_Post(EligibleViewModel viewModel)
 
         return RedirectToNextStep(nextStep, viewModel.EntryPoint);
     }
-    
+
     [HttpGet("confirmation")]
     public IActionResult Confirmation_Get()
     {
         return RedirectToAction(nameof(StaticPagesController.Index), "StaticPages");
     }
 
     [HttpGet("ineligible")]
-    public IActionResult Ineligible_Get()
+    public IActionResult Ineligible_Get(bool submitted = false)
     {
-        return RedirectToAction(nameof(StaticPagesController.Index), "StaticPages");
+        var questionnaire = questionnaireService.GetQuestionnaire();
+        var viewModel = new IneligibleViewModel
+        {
+            FoundEpcIsTooHigh = questionnaire.FoundEpcBandIsTooHigh && (questionnaire.EpcDetailsAreCorrect ?? true),

```

Comment:
This is only a small bit of logic but it would be nice to move it to the questionnaire and make use of `EffectiveEpcBand`.
In fact if we had `EffectiveEpcBandIsTooHigh` then it would simplify `IsEligibleForHug2` and close a possible loophole with expired EPCs

Reply:
Also tiny thing, but on the IneligibleViewModel I think the property name would be better as just `EpcIsTooHigh`. We use FoundEpc in other places to highlight that we haven't confirmed it's correct yet. Here we are using a confirmed EPC rating.